src/ijordan/matrixonatorTest/MatrixTest.java

```diff
@@ -12,12 +12,12 @@
 
 public class MatrixTest {
 
-	// ---------------------------------------------------
-	// MATRIX CREATION + METHODS
-	// ---------------------------------------------------
+	/*
```

Comment:
Only one comment: Changes to comments formatting like this is a little pernickity, but everything else is good.